app.js

```diff
@@ -23,6 +24,7 @@ Celebrity.init();
 //routers
 app.use(routeLogin);
 app.use(routeList);
+app.use(routeFavorite);
```

Comment:
favorite

Reply:
favourite instead of routeFavorite?

Reply:
rename the variable. it's no longer `routeFavorite`